io.github.naikari.Naikari.yaml

```
@@ -0,0 +1,117 @@
+id: io.github.naikari.Naikari
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: naikari
+finish-args:
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+
+modules:
+  - shared-modules/physfs/physfs.json
+
+  - name: luajit
+    no-autogen: true
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    cleanup:
+      - /bin
+      - /share
+    sources:
+      - type: git
+        url: https://github.com/LuaJIT/LuaJIT.git
+        branch: v2.1
+        commit: 4c2441c16ce3c4e312aaefecc6d40c4fe21de97c
```

you never put a branch.

```suggestion
        # branch: v2.1
        commit: 4c2441c16ce3c4e312aaefecc6d40c4fe21de97c
```

(I leave as a comment for information)